Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -763,16 +763,33 @@ void WebChromeClient::unavailablePluginButtonClicked(Element& element, RenderEmb
     UNUSED_PARAM(pluginUnavailabilityReason);
 }
 
-void WebChromeClient::mouseDidMoveOverElement(const HitTestResult& hitTestResult, unsigned modifierFlags, const String& toolTip, TextDirection)
+OptionSet<WebEventModifier> toWeb(OptionSet<WebCore::PlatformEventModifier> modifiers)
```

Comment:
This should be in a new file `WebEventModifier.cpp`. It's strange to declare a method in one header and then implement in an unrelated file in WebKit.

Reply:
I think I was following the wrong precedent set by `OptionSet<WebEventModifier> modifiersForNavigationAction(const NavigationAction&);` declared in the same header... but that's possibly because it's use is limited to the narrow scope of `WebChromeClient.cpp` where it's implemented.

I'll move out the function definition to `WebEventModifier.cpp` instead.